packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -206,7 +207,12 @@ class FlutterDevice {
       globals.printTrace('Connecting to service protocol: $observatoryUri');
       isWaitingForVm = true;
       vm_service.VmService service;
-
+      if (!disableDds) {
+        await device.dds.startDartDevelopmentService(
+          observatoryUri,
+          false, // TODO(bkonyi): ipv6 support?
```

Comment:
what is the status of this TODO?

Reply:
Wasn't sure where to find whether or not we were running with ipv6, but I've figured it out.